crates/aquascope/src/interpreter/mvalue.rs

```diff
@@ -454,13 +456,13 @@ impl<'tcx> Reader<'_, '_, 'tcx> {
       _ if ty.is_any_ptr() => {
         let val = self.ev.ecx.read_immediate(op)?;
         let mplace = self.ev.ecx.ref_to_mplace(&val)?;
-        if self.ev.ecx.check_mplace(&mplace).is_err() {
```

Comment:
`check_mplace` was removed, miri doesn't use it anymore so i suspect we shouldn't need it anymore. not sure what the replacement is.